aiplatform/src/main/java/aiplatform/PredictTextPromptSample.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package aiplatform;
+
+// [START aiplatform_sdk_ideation]
+
+import com.google.cloud.aiplatform.v1beta1.EndpointName;
+import com.google.cloud.aiplatform.v1beta1.PredictResponse;
+import com.google.cloud.aiplatform.v1beta1.PredictionServiceClient;
+import com.google.cloud.aiplatform.v1beta1.PredictionServiceSettings;
+import com.google.protobuf.Value;
+import com.google.protobuf.util.JsonFormat;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+// Test a text prompt with a supported large language model
+public class PredictTextPromptSample {
+
+  public static void main(String[] args) throws IOException {
+    // TODO(developer): Replace these variables before running the sample.
+    // Details of designing text prompts for supported large language models:
+    // https://cloud.google.com/vertex-ai/docs/generative-ai/text/text-overview
+    String instance =
+        "{ \"prompt\": " + "\"Give me ten interview questions for the role of program manager.\"}";
+    String parameters =
+        "{\n"
+            + "  \"temperature\": 0.2,\n"
+            + "  \"maxOutputTokens\": 256,\n"
+            + "  \"topP\": 0.95,\n"
+            + "  \"topK\": 40\n"
+            + "}";
+    String project = "YOUR_PROJECT_ID";
+    String location = "us-central1";
+    String publisher = "google";
+    String model = "text-bison@001";
+
+    predictTextPrompt(instance, parameters, project, location, publisher, model);
+  }
+
+  // Get a text prompt from a supported text model
+  public static void predictTextPrompt(
+      String instance,
+      String parameters,
+      String project,
+      String location,
+      String publisher,
+      String model)
+      throws IOException {
+    String endpoint = String.format("%s-aiplatform.googleapis.com:443", location);
+    PredictionServiceSettings predictionServiceSettings =
+        PredictionServiceSettings.newBuilder().setEndpoint(endpoint).build();
```

Comment:
Kindly chain the builder methods in a new line.

Reply:
Missed that. Changed. Thanks.

Reply:
Hmm, it turns out that the code formatter (google-java-format-1.17.0-all-deps.jar) automatically reformatted the code and overwrote the formatting change.